tests/transformers/models/test_causal_lm_models.py

```diff
@@ -35,6 +35,11 @@ def get_config(model_config):
         n_heads = config.num_key_value_heads
         d_head = config.hidden_size // config.num_attention_heads
         n_layer = 1  # config.num_hidden_layers
+    elif hasattr(config, "attn_config"):  # Check for Dbrx
```

Comment:
same, check for `d_model`

Reply:
I used it to fetch the `kv_n_heads` from `attn_config` which is different from `n_heads`. This uses GQA

Reply:
then lets search for `kv_n_heads` in json? that would be generalized little.

Reply:
I have tried that but `kv_n_heads` is not direct attribute of config. It is a key inside `attn_config` attribute. I have made changes to `get_config` in a more unique way for some models which has overlapping attributes in config. Testing for all the models locally. Will push the commit soon.

Reply:
For example, MPT and dbrx has similar `config.json`. Currently `n_heads` case is being checked for MPT. Hence current check fails, and wrong config will be set for the model. Changed it to more specific way, ` elif hasattr(config,"auto_map") and "modeling_mpt.MPTForCausalLM" in config.auto_map['AutoModelForCausalLM']`